docs/blog/microbit/2020-release.md

```diff
@@ -0,0 +1,81 @@
+https://youtu.be/1LACtv9XvXQ
+
+### Expand/Collapse Blocks
+
+We know that some power-users out there have been writing very large Block programs which can be hard to navigate. We’ve added the ability to collapse and expand clusters of blocks and Functions to help tidy the Workspace and make large programs more readable. 
+
+![Expanding and collapsing blocks](/static/blog/microbit/2020-release/expand-collapse.gif)
+
+### Python language support
+
+Some of you may know that we’ve had support for the Python programming language as well as JavaScript in MakeCode for Minecraft for a few months now (https://makecode.com/blog/minecraft/python-release). We’ve gotten a lot of positive feedback from Educators about this, so we’re excited to bring it to micro:bit in this release!
+
+https://youtu.be/G3bW_AMyDHA
+
+### Debugger
+
+Many Computing Education Researchers define knowledge of a programming language to be the mapping between syntax, semantics, and state. I think all too often we focus on syntax and semantics in our educational tools. No more! Debuggers are critical tools to help students understand the state of their program execution flow. We’ve worked hard to add debug capabilities to both Block and Text editors. 
+
+![Debugger](/static/blog/microbit/2020-release/debugger.gif)
+
+### GitHub Projects
+
+We’ve been working with our friends at GitHub over the past several months to provide a way to store MakeCode projects in a GitHub repository. This is great for group projects, version control, and teaching students about open source software development. Learn more about this feature here https://makecode.com/blog/makecode-with-github. 
+
+![GitHub](/static/blog/microbit/2020-release/github.jpg)
+
+### Web USB
+
+Could it be? A bird, a plane…? Better than superman, it’s Web USB to the rescue! No more dragging-and-dropping program files!! Web USB technology, allows you to ‘pair’ USB devices to your web browser, so when you click that Download button, it really does download directly to the micro:bit! You will need the latest firmware on your micro:bit device, and you’ll need to be using either new Edge or Chrome browsers. 
+
+![WebUSB](/static/blog/microbit/2020-release/webusb.gif)
+
+### More Music!
+
+We’ve added a couple new blocks to the Music category. Set Volume was very frequently requested by Teachers in classrooms full of micro:bits and speakers! And the Play Melody block was created by our amazing intern Kim (https://www.linkedin.com/in/kimberlymying/), and we’re excited to bring it to the micro:bit and see what lovely melodies you all create with it! 
+
+![Music](/static/blog/microbit/2020-release/music.gif)
+
+### Break & Continue
+
+These blocks can be found in the Loops category and allow you to exit out of long-running loops, or continue processing after a check or pause. 
+
+![Break and continue blocks](/static/blog/microbit/2020-release/break-continue.jpg)
+
+### Rudolph connectors
```

Comment:
Cultural reference won’t work for non-Christian users.

Reply:
Frankly, I've never heard of "dot-to-dot" being referred to as this... albeit a rather secular and western aspect of the holiday.

![image](https://user-images.githubusercontent.com/27789908/80508800-5a50cd80-892d-11ea-83a8-8dd79869c06d.png)